core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1243,6 +1247,17 @@ public static boolean tlsAllowUnsafeCiphers() {
         return TRANSIENT_SERVICE_OPTIONS;
     }
 
+    /**
+     * Returns whether default socket options defined by armeria are enabled.
```

Comment:
Don't forget the big A :laughing: 
```suggestion
     * Returns whether default socket options defined by Armeria are enabled.
```